services/app-api/src/handlers/postgres_migrate.ts

```diff
@@ -2,10 +2,6 @@ import { APIGatewayProxyHandler } from 'aws-lambda'
 import { RDS } from 'aws-sdk'
 import { execSync } from 'child_process'
 import { getDBClusterID, getPostgresURL } from './configuration'
-import {
-    migrate,
-    newDBMigrator,
-} from '../../../app-proto/protoMigrations/lib/migrator'
 
 export const main: APIGatewayProxyHandler = async () => {
```

Comment:
nit: This is outside the scope of the change but  I'm wondering about AWS lambda practices here. To me this handler could you add a better 200 return info. Right now it's very basic - not that use for logging purpose later.  I feel like it would be helpful if we returned some kind of info around which migration returned successfully.  what do you think @mojotalantikite 
```
return {
        statusCode: 200,
        body: JSON.stringify('successfully migrated'),
        headers: {
            'Access-Control-Allow-Origin': '*',
            'Access-Control-Allow-Credentials': true,
}
```

Reply:
I can definitely switch this up a bit. I'd just have to record which parts of the migrate handler ran (schema vs data) and return that. Shouldn't be that hard, happy to do it here.